README.md

```diff
@@ -106,6 +111,7 @@ In order to provide transparency on our release cycle and to maintain backward c
 
 See [the tags section of our GitHub project](https://github.com/glpi-project/java-library-glpi/tags) for changelogs for each release version. Release announcement posts on [the official Teclib' blog](http://www.teclib-edition.com/en/communities/blog-posts/) contain summaries of the most noteworthy changes made in each release.
```

Comment:
See the tags section?
Why not the release section?
Reference: https://github.com/glpi-project/java-library-glpi/releases